conans/client/tools/win.py

```diff
@@ -447,16 +446,11 @@ def vcvars_dict(settings, arch=None, compiler_version=None, force=False, filter_
     cmd = vcvars_command(settings, arch=arch,
                          compiler_version=compiler_version, force=force,
                          vcvars_ver=vcvars_ver, winsdk_version=winsdk_version, output=output)
-    cmd += " && echo __BEGINS__ && set"
-    ret = decode_text(subprocess.check_output(cmd, shell=True))
+    cmd += " && set"
```

Comment:
What if the `vcvars_command` prints something?

Reply:
The command will be `.... && set > output.tmp`, so only the output from `set` will be piped into the temp file.

Reply:
right.